src/en/models-config.md

```diff
@@ -146,7 +153,7 @@ agent-stream: devel
 ```
 
 
-## Provision machines faster by disabling software upgrades
+### APT updates and upgrades
```

Comment:
getting a shorter provision time is the ONLY circumstance you would use this. I think it is worth calling out.